src/sentry/workflow_engine/endpoints/serializers.py

```diff
@@ -122,6 +122,18 @@ def serialize(
         }
 
 
+@register(DataConditionHandler)
+class DataConditionHandlerSerializer(Serializer):
+    def serialize(self, obj: DataConditionHandler, *args, **kwargs) -> dict[str, Any]:
```

Comment:
it'd be cool if we defined this `dict[str, Any]` as a typed dict with the attributes we expect in the API contract.